integration/ksm/ksm_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
nit: it is nice to have some description at the top of the test of what the intention is, and how it is supposed to work - we have been bitten in the past when coming to look at a failing test case and not having much clue as to what it was meant to do or how ;-)